README.md

```diff
@@ -127,6 +127,25 @@ docker cp ${CONTAINER}:/root/BUILD/rpms/x86_64/ ./rpms/
 docker rm ${CONTAINER}
 ```
 
+## Building a Hotfix
+
+Sometimes it is necessary to apply a patch without rebuilding the whole release in order to
+deliver a fix quickly.  This can be accomplished by building a hotfix rpm which will apply
+a series of patches.
+
+Create a patch file using `git format-patch` and copy to `rpm_spec/patches/`.  You will have to modify the path to be relative to the RPM BUILDDIR.
+
+`docker build --pull --tag $USER/rpm_build:radjabov-hotfix .`
+
+Create a `BUILD/hotfix` directory and copy the .src.rpm for the release that you want to
+build a hotfix against, for example `manageiq-release-18.0-1.el9.src.rpm`
+
+```sh
+docker run --rm -v `pwd`/OPTIONS:/root/OPTIONS -v `pwd`/BUILD:/root/BUILD $USER/rpm_build:radjabov-hotfix build_hotfix
```

Comment:
NOTE instead of a separate volume map for the hotfix dir it seemed easier to just map `./BUILD` so the artifacts and the srpm are in one place but if this doesn't work for some reason I can change it

Reply:
I thought we already had this documented, but maybe not.  @bdunne Please review.

Reply:
Yeah I didn't see any so this started as me just taking notes on what I had to do so we had something to reference in the future

Reply:
Having the docker command here is helpful

So much time goes between needing to build rpms that I often forget.

Re: "I thought we already had this documented"

On the mac, mapping the `BUILD` volume was horribly slow.

I remember putting together documentation on this a while back but people seemed to disagree on the best approach.

Reply:
> On the mac, mapping the BUILD volume was horribly slow. Think I mapped BUILD/rpm to speed it up.
>
> I remember putting together documentation on this a while back but people seemed to disagree on the best approach.

Oh did you have hotfix documentation?  Or you mean documentation on how to build rpms on a mac?